src/components/facets/FacetRow.tsx

```diff
@@ -30,6 +30,7 @@ export class FacetRow extends React.Component<IFacetRowProps, any> {
             'facet-selectable',
             {'facet-exclude': this.props.enableExclusions},
         );
+        const labelClassName = this.props.isChecked && this.isExclude ? ' text-exclude' : '';

```

Comment:
It would be more standard to do it like this:
```
const labelClassName = classNames({
  'text-exclude': this.props.isChecked && this.isExclude
})
```